tests/test_model_manual.py

```diff
@@ -89,7 +89,7 @@ def test_valid_model(tmp_dir, func, name, expectation):
 def test_compare_get_model_new_and_old():
     m_new = get_model_new(parfile)
     m_old = get_model(parfile)
-
+    print(parfile)
```

Comment:
Do not print. You can always run `pytest --pdb` and inspect local variables in a failing test.

Reply:
I don't understand the "Do not print."  Why is that?  It gets suppressed during normal usage, but you can see if if you want by using `pytest -s`.  It seems (and has been) useful to me.